### PR TITLE
Yii usage about CRSF

### PR DESCRIPTION
**Pull request by Shnoulle**
Yii already have a good CRSF system : we just have to control if it's a POST request

An,d jquery ajax already have CRSF token adden in whole ajax request : no need to make a new hack again and again.

Question : what is the usage of _getPostOrParam : i really think we must just use getPost (if we need POST) or getParam if we allow GET or POST

### CODE REVIEW

**Comment by Shnoulle** [comment]
Child of https://github.com/LimeSurvey/LimeSurvey/commit/21955942ad58dfb3907af76033070bd1ca3cf53b

File: application/controllers/admin/useraction.php [hunk 1]
```diff
@@ -177,93 +180,91 @@ public function adduser()
     */
     public function deluser()
     {
+        if(Yii::app()->request->getIsPostRequest()){ /* DB action : need post request */
```
Comment (Shnoulle) [inline application/controllers/admin/useraction.php]: I just add this test : all other red/green line is about space

**Comment by LouisGac** [comment]
you can (must ^^) use the Yii Crsf in Ajax context

**Comment by Shnoulle** [comment]
1. Yii::app()->request->getIsPostRequest() force to be $_POST
2. We already use CRSF if we use ajax/post : no need extra solution here, we do it in 2.00 : but need to be a POST

**Comment by Shnoulle** [comment]
@lacrioque can we merge this one ?

**Pull request merged**